src/overlays/ovl_i1/DFEE30.c

```diff
@@ -3,7 +3,25 @@
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/DFEE30/func_i1_801920F0.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/DFEE30/func_i1_801922DC.s")
+extern s32 D_i1_8019C0B8;
```

Comment:
Can you move this extern to the top of the file after the includes?

Reply:
Sure!